default.nix

```diff
@@ -1,14 +1,12 @@
 {pkgs ? import <nixpkgs> {}}:
-pkgs.rustPlatform.buildRustPackage rec {
```

Comment:
`rec` is needed when you reference same scope variable multiple times, in here it's not needed